services/wfm/src/main/java/org/openkilda/wfm/topology/flow/bolts/CrudBolt.java

```diff
@@ -523,6 +529,26 @@ private void handleReadRequest(String flowId, CommandMessage message, Tuple tupl
         outputCollector.emit(StreamType.RESPONSE.toString(), tuple, values);
     }
 
+    private void handleMeterModeRequest(CommandMessage inMessage, Tuple tuple, final String flowId) {
+        FlowPair flowPair = flowService.getFlowPair(flowId)
+                .orElseThrow(() -> new MessageException(inMessage.getCorrelationId(), System.currentTimeMillis(),
+                        ErrorType.NOT_FOUND, "Can not get flow", String.format("Flow %s not found", flowId)));
+
+        SwitchId switchId = flowPair.getForward().getSrcSwitch().getSwitchId();
+        long bandwidth = flowPair.getForward().getBandwidth();
+        Integer meterId = flowPair.getForward().getMeterId();
```

Comment:
Shouldn't we update meter on the dst switch, too?

Reply:
Roman is right, We need update meter on last switch too. 
flowPair.getReverse().getSrcSwitch()

Reply:
Update